server/src/com/cloud/network/vpn/Site2SiteVpnManagerImpl.java

```diff
@@ -438,9 +438,6 @@ public Site2SiteCustomerGateway updateCustomerGateway(UpdateVpnCustomerGatewayCm
             name = "VPN-" + gatewayIp;
         }
         String guestCidrList = cmd.getGuestCidrList();
-        if (!NetUtils.validateGuestCidrList(guestCidrList)) {
```

Comment:
@mrunalinikankariya  we need to check for the isValidCIDR cidr here. If you look at validateGuestCidr method it is checking for the isValidCIDR first then checking for RFC compliant.

Also please squash the commits into single commit.